### PR TITLE
additional eslint settings

### LINKED ISSUE #45: Use ES6, or keep IE11 support?
There are a lot of useful features in es6 that are supported natively by most browsers except for IE11.

Do we want to stop supporting IE11 and use these new features?

### PR DESCRIPTION
**Pull request by Zeragamba**
[fixes #45]

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -1,13 +1,20 @@
...
+        "curly": ["error"],
+        "indent": ["error", 4],
         "indent": [ "error", 4 ],
         "linebreak-style": [ "error", "windows" ],
         "semi": [ "error", "always" ],
         "eol-last": [ "error", "always" ],
+        "eqeqeq": ["error"],
+        "no-trailing-spaces": ["error"],
     }
 };
```
Comment (camerongillette) [inline .eslintrc.js]: Can you add a newline here?

**Comment by camerongillette** [comment]
+1 CR

**Review by camerongillette — APPROVED** [review]
This looks good, does it accommodate the error we're getting the error we're getting on PR #43 where 'class' is a reserved word?

**Comment by Zeragamba** [comment]
yep, classes are an es6 feature, it should resolve it

**Pull request merged**